.github/workflows/docs.yaml

```diff
@@ -37,9 +37,15 @@ jobs:
       - name: Get the current Synapse version
         id: vars
```

Comment:
I think you'll need a `shell: bash` to do the bash magic?

Reply:
Hmm, good point. Turns out this works in POSIX shells as well. Will edit the wording.